### ISSUE TITLE
Issue with capital letter in usernames selected from the dropdown popup window

### ISSUE THREAD

**Issue by Eldenroot** [WildcardSearch/MentionMe#168]
There is a small issue with usernames which contains "capital letter".

For example: user "**C**hmura" has capital letter "**C**" at the beginning of his username.

When you type "@" and then first letter (normal "**c**" is used) "**c**" -> popup window is shown -> select "**Chmura**" but in the editor there is "**c**hmura" shown instead of "**C**hmura" with capital letter.

Anyway when you post the thread/post -> output is shown right with capital letter at the beginning (on full edit or edit there is a normal "**c**" instead).

It is a minor bug, mainly a visual glitch.

**Comment by WildcardSearch** [comment]
Thanks for the report.

Is there no issue when there are capital letters in the rest of the user name? Like `myUsername`?

**Comment by Eldenroot** [comment]
In my opinion - when you select anything from the popup dropdown window, the content should be replaced with the selected username from the list completely, not just autocompleted

**Comment by WildcardSearch** [comment]
Idon't understand what you mean-- can you rephrase:?

**Comment by Eldenroot** [comment]
I am sorry, I will try to explain this issue in a better way.

1) type "@"and then for example "c" -> dropdown window is opened
2) there is a list of available username starting with the letter "c"
3) choose a user with the username beginning with capital "C" (in that case user "Chmura"), not the "normal"one
4) check out the output -> mentioned user is showed as "**c**hmura", not as "**C**hmura" in the text editor because you type the first letter "c" and the rest of the username was added automatically. 

In my opinion when you choose somebody from the dropdown menu, the whole content after "@" should be replaced by the selected username.

Is it better?

**Comment by WildcardSearch** [comment]
Yes. Thank you.

**Comment by WildcardSearch** [comment]
Since this is very minor, and not really a bug, I'm gonna bump it to 3.2 so I can go ahead and get started rewriting the JavaScript and not have to worry about doing a release for this small change.

**Comment by Eldenroot** [comment]
OK, it converts automatically all capital letters into the normal ones - Eldenroot -> eldenroot etc.

**Comment by WildcardSearch** [comment]
>it converts automatically all capital letters into the normal ones - Eldenroot -> eldenroot etc.

Since the update?

**Comment by Eldenroot** [comment]
I am not sure :(

**Comment by WildcardSearch** [comment]
It shouldn't. I just pushed a fix here: d66ed9054b3171d4107474454f9c1063d2072842

**Comment by Eldenroot** [comment]
@WildcardSearch - tested your fix, this issue is fixed, thank you!
